fundamental.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+
+def get_financials(url):   
+    page_source = requests.get(url)
+
+    soup_html = BeautifulSoup(page_source.text, 'html.parser')
+    table_soup = soup_html.find('div', class_="M(0) Whs(n) BdEnd Bdc($seperatorColor) D(itb)")
+    
+    # Get Title Row
+    headlines = table_soup.find('div', class_="D(tbr) C($primaryColor)")
+    headlines = headlines.findAll('span')
+    title_row = list()
+    for line in headlines:
+        title_row.append(line.text)
+    
+    table = pd.DataFrame(None, columns = title_row)
+    remaining_lines = table_soup.findAll('div', class_="D(tbr) fi-row Bgc($hoverBgColor):h")
+    for row in remaining_lines:
+        columns = row.findChildren('div',recursive=False)
+        line_values = list()
+        for col in columns:
+            if ',' in col.text:
+                value = float(col.text.replace(',', ''))
+            elif '-' in col.text and len(col.text) == 1:
+                value = np.nan
+            else:
+                value = col.text
+            line_values.append(value)
```

Comment:
Uma coisa legal é deixar o condicionais dos ifs legível

```suggestion
            hasComma = ',' in col.text
            isEmpty = '-' in col.text and len(col.text) == 1
            if  hasComma :
                value = float(col.text.replace(',', ''))
            elif isEmpty:
                value = np.nan
            else:
                value = col.text
            line_values.append(value)
```

Reply:
check